app/models/block/hero.rb

```diff
@@ -0,0 +1,21 @@
+module Block
+  HeroImageSources = Data.define(:desktop, :desktop_2x, :tablet, :tablet_2x, :mobile, :mobile_2x)
+  HeroImage = Data.define(:alt, :sources)
+
+  class Hero < Block::Base
+    attr_reader :image, :hero_content
+
+    def initialize(block_hash)
+      super(block_hash)
+
+      alt, sources = block_hash.fetch("image").values_at("alt", "sources")
```

Comment:
Minor, and not a blocker, but `data` could be used from this point onwards rather than `block_hash`.

Reply:
13727cdee3ffa59f128e27af893c6d0429d6482f